### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by developertariq**
# Highlights
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- Filtered the data that retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project deployed and accessible online (add a link to the online version of your app to your README file).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,8 +206,8 @@ npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standa
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
-
-- **Add, edit and display category**
+- Implement some UX improvements: include transitions and/or animations, etc.
+- Implement additional pages in the website (each with a route in the SPA): about me, references, etc.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (GzimAsani) [inline README.md:213]: - [ ] Under the "Acknowledgements" section please give  appropriate credit to the designer

File: src/_test_/Continent.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { BrowserRouter } from 'react-router-dom';
+import store from '../Redux/configureStore';
+import '@testing-library/jest-dom';
+import Continent from '../components/Continent';
+
+describe('Continent component', () => {
+  test('renders correctly', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
```
Comment (GzimAsani) [inline src/_test_/Continent.test.js:14]: - [ ] Good job adding tests on the correct render of your pages but please add also unit tests for several components of the app and some pure functions in the Redux code.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @developertariq                :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the redux and styling :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @developertariq     :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional README file :heavy_check_mark: 
- Professional PR description :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**